canvas_modules/harness/package.json

```diff
@@ -5,8 +5,10 @@
   "license": "Apache-2.0",
   "scripts": {
     "start": "NODE_ENV=development node index.js",
+    "cypress:run": "cypress open",
```

Comment:
Why did we add this?  We already run a similar cmd under `test`.

Reply:
I believe this was an extra script, so I went ahead and removed it to eliminate redundancy.